compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.cpp

```diff
@@ -364,6 +364,91 @@ ValueRange DispatchRegionOp::getResultDynamicDims(unsigned idx) {
                                shapedType ? shapedType.getNumDynamicDims() : 0);
 }
 
+/// Canonicalizes a DispatchRegionOp: Drop all unused results.
+bool dropUnusedDispatchRegionResults(RewriterBase &rewriter,
+                                     Flow::DispatchRegionOp &regionOp) {
+  assert(regionOp.getBody().hasOneBlock() &&
```

Comment:
If this is part of the verifier of `DispatchRegionOp` then we dont need this here.

Reply:
added to the verifier